<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiBike</class>
 <widget class="QMainWindow" name="PiBike">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabData">
       <attribute name="title">
        <string>Live Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0">
        <item row="0" column="0">
         <widget class="QLabel" name="lblDist">
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Distance</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lblAvgSpeed">
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Avg Speed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="lblCad">
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cadence</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblAcc">
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Acceleration</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblElev">
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Elevation Gain</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblSpeed">
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Speed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lblElevVal">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblDistVal">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblSpeedVal">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lblAvgSpeedVal">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lblAccVal">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="lblCadVAl">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGraphs">
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabMao">
       <attribute name="title">
        <string>Map</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
